Assets.xcassets/Calculator.imageset/Contents.json

```diff
@@ -0,0 +1,62 @@
+{
+  "images" : [
+    {
+      "filename" : "Calculator-light-1x.png",
+      "idiom" : "mac",
+      "scale" : "1x"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "light"
+        }
+      ],
+      "filename" : "Calculator-light-1x-1.png",
+      "idiom" : "mac",
+      "scale" : "1x"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "filename" : "Calculator-dark-1x.png",
```

Comment:
[Template images](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/custom-icons/) allow to use the same image file for both dark and light icon.
(Not necessary to fix, just for information)

Reply:
I didn't knew this! Thanks very much.

Reply:
Thanks for info. I'll change it to the template image.

Reply:
Fixed in jnooree/RDM@e99669b1727cc5fe6508a1655f46f4a6c78bb050.

Reply:
You forgot to add the new image file.

Reply:
Sorry 😅 Now it is really fixed in jnooree/RDM@e4102bc!